magma/lib/magma.rb

```diff
@@ -134,6 +134,19 @@ def find_descendents(klass)
     end
   end
 
+  def setup_yabeda
+    Yabeda.configure do
+      group :magma do
+        gauge :data_rows do
+          comment "The number of data rows by project and model for magma."
```

Comment:
We should keep in mind (when communicating to users) that this might be a rough number because of disconnected or otherwise "bad quality" records. So it's not a "QC passed" number of records.